### PR TITLE
ref(replays): link mutation breadcrumb to the docs

### LINKED ISSUE #51656: Replay: Mutation breadcrumb should lead to the docs
When the Sentry SDK detects multiple mutations, it creates a breadcrumb to warn the developer.

image

That's documented here: https://docs.sentry.io/platforms/javascript/session-replay/configuration/#mutation-limits

Additionally, we documented in our troubleshooting page (https://docs.sentry.io/platforms/javascript/session-replay/troubleshooting/#i-see-the-message-a-large-number-of-mutations-was-detected-n-this-can-slow-down-the-replay-sdk-and-impact-your-customers) this crumb. In the event a confused user looks it up.

This workflows assumes a confused user will look it up (google it or search our docs). We believe this crumb albeit valuable can be confusing to users ("what should I do") hence we should be linking to the docs directly from the breadcrumb.

### PR DESCRIPTION
**Pull request by michellewzhang**
closes [#51656](https://github.com/getsentry/sentry/issues/51656)
SCR-20231031-jpyp

### CODE REVIEW

**Review by ryan953 — DISMISSED** [review]
This is highjacking the usual click behavior, so clicking on the timestamp for example doesn't take you to that point in time in the replay anymore :(

Maybe we can update the description or body of the crumb to have a link in it. I'm thinking about a similar if-statement inside the render method of `static/app/components/replays/breadcrumbs/breadcrumbItem.tsx` thats like `<a href="..." external>Read the docs</a>`

**Comment by billyvg** [comment]
The title is also for generic Replay issues and not mutation specific (e.g. in future we could add other crumbs with `Replay` as title)

**Comment by michellewzhang** [comment]
@billyvg @ryan953 changes made!

File: static/app/utils/replays/getFrameDetails.tsx [hunk 1]
```diff
@@ -124,13 +125,27 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
...
-      ? t(
-          'A large number of mutations was detected (%s). Replay is now stopped to prevent poor performance for your customer.',
-          frame.data.count
+      ? tct(
+          'A large number of mutations was detected [count]. Replay is now stopped to prevent poor performance for your customer. [link]',
+          {
+            count: frame.data.count,
+            link: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/session-replay/configuration/#mutation-limits">
+                {t('Read the docs.')}
```
Comment (billyvg) [inline static/app/utils/replays/getFrameDetails.tsx:134]: "Read the docs" seems a bit too "RTFM" -- what about something like "Learn more" or "Learn more from our docs" or "Read more from our docs"

**Review by billyvg — APPROVED** [review]
Are there any issues with moving from the row to the tooltip? (i.e. does tooltip disappear when you go to click the link?)

File: static/app/utils/replays/getFrameDetails.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/utils/replays/getFrameDetails.tsx:134]: the tooltip is actually the content in the row but overflowed!

Comment (michellewzhang) [reply static/app/utils/replays/getFrameDetails.tsx:134]: and i added the `isHoverable` property to the tooltip

**Pull request merged**